Examples/Sample_xDscWebServiceRegistrationWithSecurityBestPractices.ps1

```diff
@@ -53,7 +53,7 @@ configuration Sample_xDscWebServiceRegistrationWithEnhancedSecurity
             DependsOn               = "[WindowsFeature]DSCServiceFeature" 
             RegistrationKeyPath     = "$env:PROGRAMFILES\WindowsPowerShell\DscService"   
             AcceptSelfSignedCertificates = $true
-            UseUpToDateSecuritySettings = $true
+            UseSecurityBestPractices = $true
```

Comment:
Yes, for some specific settings, it is recoverable. But say, for a configuration file who's content is reset to desired content, it is not recoverable to its original content. Same for registry. And here, we are actually setting registry to use security best practice.